device-types/Arista/DCS-7280CR3-96.yaml

```diff
@@ -0,0 +1,216 @@
+---
+manufacturer: Arista
+model: DCS-7280CR3-96
+slug: arista-dcs-7280cr3-96
+part_number: DCS-7280CR3-96
+u_height: 2
+is_full_depth: true
```

Comment:
Please add the `airflow` definition if you have or can find it. This is often available in the products datasheet, but sometimes it is not. If you cannot find it just let us know and resolve this comment.

Reply:
@danner26 Arista has 3 different SKU's (the one mostly used, all but one in this repo is without airflow).
Do you want me to change that in this PR to use the airflow SKU's instead which ends with -R or -F